v_4_7_0/files/k8s-resource/calico-all.yaml

```diff
@@ -460,7 +460,7 @@ spec:
               memory: 150Mi
             limits:
               cpu: 250m
-              memory: 150Mi
+              memory: 500Mi
```

Comment:
Was that introduced in some previous commit?

Reply:
No, this was a leftover from my earlier "reset all" command. PM currently on-hold in our team.